compiler/noirc_frontend/src/lexer/lexer.rs

```diff
@@ -704,7 +720,7 @@ mod tests {
 
     #[test]
     fn test_single_double_char() {
-        let input = "! != + ( ) { } [ ] | , ; : :: < <= > >= & - -> . .. % / * = == << >>";
+        let input = "! != + ( ) { } [ ] | , ; : :: < <= > >= & - -> . .. ..= % / * = == << >>";
```

Comment:
The name of this test now slightly doesn't match because of this, but it's not a big deal (feel free to choose a different name if you want, or leave it as is)